app/controllers/api/v1/invoices/invoice_customer_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Invoices::InvoiceCustomerController < ApplicationController
```

Comment:
@Dpalazzari : Now I'm thinking our namespaced controllers could save redundancy by taking out the parent namespace from the controller name. In this case, `Api::V1::Invoices::CustomerController`